src/main/java/org/mineacademy/fo/model/SimpleHologramStand.java

```diff
@@ -33,35 +33,34 @@ public class SimpleHologramStand extends SimpleHologram {
 	 * Create a new simple hologram using armor stand showing the given material
 	 *
 	 * @param spawnLocation
-	 * @param material
+	 * @param item
 	 */
-	public SimpleHologramStand(Location spawnLocation, CompMaterial material) {
```

Comment:
Please do not make breaking changes, students watching training videos will get compile issues if the constructor is changes. Can you recode this fix to retain the old CompMaterial constructor, or simply create another one with ItemStack, keeping the old one instead?